components/auth/AllowedPrincipals.vue

```diff
@@ -76,6 +76,7 @@ export default {
         />
       </div>
       <div class="col span-6">
+        <h4 v-if="accessMode!=='unrestricted'" class="mb-20"><t k="authConfig.allowedPrincipalIds.title'" :raw="true" /></h4>
```

Comment:
There's something a little disjointing with this header (in respect to the `Configure who... ` one on the left). Reducing the margin down to mb-10 helps to glue it to the content. I also looked at adding a matching header to the radio buttons ... but adding it just to balance out the sections isn't great.

Reply:
Ok, my-10 is the default for h4 so I’ll remove the extra class